local/node.go

```diff
@@ -38,27 +38,50 @@ type getConnFunc func(context.Context, node.Node) (net.Conn, error)
 // AvalancheGo binaries in tests
 type NodeProcess interface {
```

Comment:
We should document the behavior better...namely:

1. What happens if you call Start/Stop multiple times? Or out of order?
2. What happens if you call Wait and Stop had already been called?
3. What happens if you call Alive but Start hasn't been called?